Source/Core/CommandLineOptions.cs

```diff
@@ -2012,6 +2022,7 @@ public override void Usage()
   Multiple .bpl files supplied on the command line are concatenated into one
   Boogie program.
 
+  /lib           : Include definitions in LibraryDefinitions.bpl
```

Comment:
I would not mention the file name `LibraryDefinitions.bpl` here, because it is something internal.